spyql/parser.py

```diff
@@ -269,7 +277,7 @@ def parse_fromto(clause, strings, formats_list):
     return res
 
 
-def make_expr_ready(expr, strings):
+def make_expr_ready(expr):
```

Comment:
🙃 Nitpick: perhaps in the future changes like these, where we're only cleaning up things not related to the feature itself, can be in a separate commit.
Mostly because it feels strange having these changes described by a message saying "added universal dot operator"